src/basics/BasicsValidator.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace basics;
+class BasicsValidator implements BasicsValidatorInterface
+{
+    public function isMinutesException(int $minute):
```

Comment:
Please realign this file. All functions should be on same level of 4 spaces.
Please write functions like this:

public function isYearException(int $year): void
{